portfolio/src/main/webapp/script.js

```diff
@@ -16,24 +16,33 @@
 
 $(document).ready(function() {
 
+    // Create the text for the copyright information
     let date = new Date();
     let year = date.getFullYear();
     $("#copyright").html(`&copy; ${year} Darren Thompson`);
 
+    // Use the height of the header/navigation to create an
+    // offset of the main content
     let $height = $("header").outerHeight(true);
-
     $("#main-container").css("margin-top", `${$height + 30 + "px"}`);
 
+
+    // Calculate the threshold for the scroll dialog
     let threshold = $(document).height() * 0.1;
     let didShowDialog = false;
 
+    // When the user crosses the specified threshold
+    // show a dialog with a link to my personal site. 
     $(window).scroll(function() {
         if($(window).scrollTop() + $(window).height() > $(document).height() - threshold && !didShowDialog) {
             $("#dialog").show();
+            // Additionally the boolean is used to make sure
+            // the dialog is only shown once
             didShowDialog = true;
         }
     });
 
+    // Hide the dialog window when the website link is clicked
     $("#message a").click(function(event) {
         $("#dialog").hide();
     });
```

Comment:
You don't need to make this change as part of this commit, but you might consider also hiding the dialog when the user clicks outside of it, so they can go back to appreciating this website. :)